working-with-distributed-workloads.adoc

```diff
@@ -25,6 +25,8 @@ This approach significantly reduces the task completion time, and enables the us
 include::modules/overview-of-distributed-workloads.adoc[leveloffset=+1]
 include::modules/overview-of-kueue-resources.adoc[leveloffset=+2]
 
+include::assemblies/managing-custom-training-images.adoc[leveloffset=+1]
+
```

Comment:
Should the modules/pushing-an-image-to-the-integrated-openshift-image-registry.adoc file be added there as well?

Reply:
It's added within that assembly file, see line 12.